docs/configuration.md

```diff
@@ -0,0 +1,87 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution uses a variety of linters under the hood without exposing
+implementation details and complexity of the underlying tools to the user. Because of that,
```

Comment:
Run-on sentence beginning with "Because of that,".   Also, grammatically you shouldn't start a sentence with Because.

Reply:
Thanks for your grammar advice!